platform/ios/src/MGLMapViewDelegate.h

```diff
@@ -17,7 +17,19 @@ NS_ASSUME_NONNULL_BEGIN
  individual annotations.
  */
 @protocol MGLMapViewDelegate <NSObject>
-
+#if __IPHONE_OS_VERSION_MAX_ALLOWED >= 140000
+/**
+ Tells the delegate that the map's location updates accuracy authorization has changed.
+ 
+ This method is called after the user changes location accuracy authorization when
+ requesting location permissions or in privacy settings.
+ 
+ @param mapView The map view that changed its location accuracy authorization.
+ @param accuracyAuthorization The new accuracy authorization value.
+ 
+ */
+- (void)mapView:(MGLMapView *)mapView didChangeAccuracyAuthorization:(CLAccuracyAuthorization)accuracyAuthorization;
+#endif
```

Comment:
We shouldn't need the xcode 11 compatibility test here as this is just a wrapper.

Also -- shouldn't this method be optional? There can be consumers of the map who don't care about user location entirely.

Reply:
> We shouldn't need the xcode 11 compatibility test here as this is just a wrapper.

In order to build the SDK with either Xcode 11/12 this has to be here. Unless we drop Xcode 11 support.

> Also -- shouldn't this method be optional? There can be consumers of the map who don't care about user location entirely.

Right. However with the privacy changes this has to be required to let developers be aware that they will have to handle that, even tho they don't use location updates.